components/symbl_ai/actions/post-video-summary-ui/post-video-summary-ui.mjs

```diff
@@ -0,0 +1,114 @@
+import symblAIApp from "../../symbl_ai.app.mjs";
+
+export default {
+  key: "symbl_ai-post-video-summary-ui",
+  name: "Submit Video Summary User Interface",
+  description: "The Video Summary UI provides users the ability to interact with the Symbl elements (transcripts, Questions, Follow-Ups, Action Items, etc.) from a video conversation. See the doc [here](https://docs.symbl.ai/docs/async-api/overview/video/post-video-url)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    symblAIApp,
+    conversationId: {
+      propDefinition: [
+        symblAIApp,
+        "conversationId",
+      ],
+    },
+    videoUrl: {
+      type: "string",
+      label: "Video URL",
+      description: "URL of the video file for which you want to generate the video summary UI.",
+    },
+    name: {
+      type: "string",
+      label: "Name",
+      description: "Use `video-summary` for the Video Summary UI.",
+      default: "video-summary",
+    },
```

Comment:
It looks like this field can only be "video-summary", "verbose-text-summary", or "audio-summary". We could list these as options for the user to select from, but since this component is meant for posting video summaries, I think this prop could be removed and hard coded in `data`.
```suggestion
```

Reply:
Thanks @michelle0927. This might actually change in the future but I agree with you that's not currently necessary. Fixed in d2cf63aa.